sympy/physics/mechanics/system.py

```diff
@@ -791,7 +790,7 @@ def get_joint(self, name):
                 return joint
 
     def _form_eoms(self):
-        self.form_eoms()
+        return self.form_eoms()
```

Comment:
It still feels odd to me that we have a "private" method that's calling a "public" method of the same name. Why not just called `self.form_eoms` directly and save the function indirects? Or are you trying to have the private version cache the result for efficiency? In which case use `functools`.

Reply:
No, `_form_eoms` is an `abstractmethod` in `_Methods`, so it has to be implemented. It is just entirely useless.